### PR TITLE
Merge Dev into Master

### PR DESCRIPTION
**Pull request by antonelepfl**

### CODE REVIEW

**Comment by alex4200** [comment]
Tested on STAGE, seems to work

**Comment by alex4200** [comment]
Changes testes on STAGE, can be merged to PROD

File: src/assets/config_files/usecases.json [hunk 1]
```diff
@@ -1145,6 +1151,45 @@
...
+            "img": "https://raw.githubusercontent.com/antonelepfl/usecases/dev/src/assets/images/reduction.png",
+            "disabled": false,
+            "files": [
+              {
+                "entryname": "Circuit reduction",
+                "appid": 175,
+                "contenttype": "x-ipynb+json",
+                "extension": ".ipynb",
+                "file": "3b26e963-4cd2-40cd-b889-95cffb74d781",
+                "file_prod" :""
```
Comment (antonelepfl) [inline src/assets/config_files/usecases.json:1188]: @alex4200 to push to prod, we need the notebook on GIthub (production_notebooks folder)

**Comment by antonelepfl** [comment]
Merged to PROD

**Pull request merged**